src/BuildQueue.cpp

```diff
@@ -1434,23 +242,21 @@ namespace t2
     int32_t   *build_queue = queue->m_Queue;
     NodeState *node_states = queue->m_Config.m_NodeState;
 
+    int amountQueued = 0;
     for (int i = 0; i < count; ++i)
     {
       NodeState* state = node_states + start_index + i;
 
-      NodeStateFlagQueued(state);
-
-      // Verify node hasn't been touched already
-      CHECK(state->m_Progress == BuildProgress::kInitial);
```

Comment:
was this check omitted intentionally? i can't think of a reason why this pr would change its validity

Reply:
I've removed the complete concept of BuildProgress.  there is no more progress that goes from one state to the next.   nodes get processed atomically from "not yet processed"  to "processed".